alphadia/utilities/fragment_container.py

```diff
@@ -0,0 +1,130 @@
+import numba as nb
```

Comment:
I think the naming is a bit strange to have both `utils.py` and `utilities/*`.
Do you think this could be changed?